src/frontend/js/chat.js

```diff
@@ -190,73 +181,105 @@ export class Chat extends LitElement {
+      }
+      to {
+        transform: rotate(360deg);
+      }
+    }
+
+    #settingsIcon:hover {
+      animation: rotationAnim 2s infinite linear;
+    }
 
-  `
+    :not(.dropdown)::-webkit-scrollbar {
+      background-color: #0074bc;
+      border-radius: 10px;
+      border: 5px solid #003366;
+    }
+
+    :not(.dropdown)::-webkit-scrollbar-thumb {
+      background-color: #0da2ff;
+      border-radius: 10px;
+      width: 5px;
+      border: 3px solid #003366;
+    }
+  `;
 
   render() {
     //aggiungo il main e lo metto in absolute per non andare in display flex che avevo messo per il login
     return html`
-      <main> 
-
-        <header>
-          <div id="searchChats">
-              <input type="text" placeholder="cerca farmacie" \>
-              <span> <i class="fa-solid fa-magnifying-glass"></i> </span>
-          </div>
-        </header>
+      <main>
+        <header></header>
 
         <section>
+          <div class="conversazioni">
 
-            <div class="conversazioni">
-              <div>
-                <div class="avatar"></div>
-                <p class="name">farmacia1</p>
-              </div>       
-              
-              <div>
-                <div class="avatar"></div>
-                <p class="name">farmacia2</p>
-              </div>       
-              
-              <div>
-                <div class="avatar"></div>
-                <p class="name">farmacia3</p>
-              </div>       
-              
-            </div>
+            <il-search></il-search>
+            <il-chats></il-chats>          
+            
+          </div>
 
           <div class="chat">
-
             <div class="chatHeader">
-              <h2>ChatBox ${this.login.username}</h2>
-            </div>
+              <div class="settings">
+                <span class="material-icons" id="settingsIcon">settings</span>
+              </div>
 
-              <ul class="messageBox">
-                ${this.messages.map((item, _) => html` <li>${item}</li> `)}
-              </ul>
-
-              <div id="inputControls">
-                  <input
-                  type="text"
-                  placeholder="Scrivi un messaggio..."
-                  @input=${this.onMessageInput}
-                  .value=${this.message}
-                />
-                <div class="submitContainer">
-                  <button @click=${this.sendMessage}> &gt; </button>
-                </div>
-             </div> 
+              <div class="contatto">
+                <h2>ChatBox ${this.login.username}</h2>
+              </div>
+            </div>
 
+            <ul class="messageBox">
+              ${this.messages.map((item, _) => html` <li>${item}</li> `)}
+            </ul>
+
+            <div id="inputControls">
+              <input
+                type="text"
+                placeholder="Scrivi un messaggio..."
+                @input=${this.onMessageInput}
+                @keydown=${this.controlloTastoInvio}
```

Comment:
usiamo l'inglese per i nomi di classi, funzioni o variabili

Reply:
cambiato in checkEnterKey